tests/test_views.py

```diff
@@ -4701,5 +4702,193 @@ def test_dashboard_institutions(self):
         assert_not_equal(dashboard_institutions[0]['id'], self.inst_four._id)
         assert_not_equal(dashboard_institutions[0]['id'], self.inst_five._id)
 
+
+class TestConfirmationViewBlockBingPreview(OsfTestCase):
```

Comment:
I decide to put all BingPreview blocking tests together in one class `TestConfirmationViewBlockBingPreview` in `test/test_views.py` since tests for `reset_password_get`, `confirm_email_get`, `external_login_confirm_email_get`, `claim_user_form` and `claim_user_registered` are scattered in different places.